packages/amplify-graphql-index-transformer/src/utils.ts

```diff
@@ -26,7 +27,11 @@ export function lookupResolverName(config: PrimaryKeyDirectiveConfiguration, ctx
   }
 
   if (!resolverName) {
-    resolverName = `${op}${object.name.value}${op === 'list' ? 's' : ''}`;
+    if (op === 'list') {
+      resolverName = `${op}${plurality(object.name.value, true)}`;
+    } else {
+      resolverName = `${op}${object.name.value}`;
+    }
   }
```

Comment:
This looks like a breaking change in the way, more details in 
before: 
- `type Wish` generates `listWishs` 
- `type Wishes` generates `listWishess`

after:
- `type Wish` generates `listWishes` 
- `type Wishes` generates `listWishes`

Reply:
what is the current logic for generating the resolverName for conflict resolution enabled APIs for  sync queries?